R/rix.R

```diff
@@ -156,7 +156,7 @@ rix <- function(r_ver = "latest",
                 local_r_pkgs = NULL,
                 tex_pkgs = NULL,
                 ide = c("other", "code", "radian", "rstudio", "rserver"),
-                project_path,
+                project_path = getwd(),
```

Comment:
That’s not going to fly:

>Please ensure that your functions do not write by default or in your examples/vignettes/tests in the user's home filespace (including the package directory and getwd()). This is not allowed by CRAN policies.
**Please omit any default path in writing functions**. In your
examples/vignettes/tests you can write to tempdir().
-> R/rix.R; R/rix_init.R

it’s ok for nix_build() because it’s not writing, only looking for a default.nix